api/test/integration/env/configurations/cluster/manager/configuration_files/entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
 mkdir -p /var/ossec/stats/totals/2019/Aug/
-cp -rf /tmp/configuration_files/ossec-totals-27.log /var/ossec/stats/totals/2019/Aug/ossec-totals-27.log
-chown -R wazuh:wazuh /var/ossec/stats/totals/2019/Aug/ossec-totals-27.log
+cp -rf /tmp/configuration_files/wazuh-totals-27.log /var/ossec/stats/totals/2019/Aug/wazuh-totals-27.log
```

Comment:
????

Reply:
Conflict that wasn't solver properly, reviewed